R-packages/evalcast/R/get_predictions.R

```diff
@@ -109,7 +115,8 @@ get_predictions_single_date <- function(forecaster,
            ahead = ahead[i],
            geo_type = geo_type,
            geo_values = geo_values,
-           from_covidhub = FALSE)
+           from_covidhub = FALSE,
+           ...)
```

Comment:
I'm wondering if it would be better to have an attribute called `additional_arguments_to_forecaster` (or something shorter than that) and then change the line to `additional_arguments_to_forecaster = list(...))`.  What do you think?

Reply:
Sure, sounds good.  I used `forecaster_params = list(...)`.